### ISSUE TITLE
Mehr Links hinzufügen

### ISSUE THREAD

**Issue by YourMJK** [YourMJK/dreimetadaten#28]
**Zusätzliche URLs für jede Folge:**
* Offizielle dreifragezeichen.de-Produktseite
* Streaming-Dienst (Apple Music, Spotify, Prime, …)
* iTunes
* Amazon
* …

**Comment by stehefan** [comment]
Ich würde gerne auch die YouTube Music Links hinzufügen, ein PR kann ich stellen mit allen notwendigen Änderungen sofern das für dich passt.

**Comment by YourMJK** [comment]
@stehefan  
Gerne, das würde mich sehr freuen!

Hättest du in deinem PR die Daten gleich in der db.sql-Datei?

**Comment by stehefan** [comment]
Genau, wäre der Plan die Modelle und Co. anzupassen, inkl. der DB. 😊

**Comment by stehefan** [comment]
PR ist hinzugefügt! #79
